cmd/soroban-cli/src/wasm.rs

```diff
@@ -28,6 +35,18 @@ pub enum Error {
     Parser(#[from] wasmparser::BinaryReaderError),
     #[error(transparent)]
     ContractSpec(#[from] contract::Error),
+
+    #[error(transparent)]
+    Locator(#[from] locator::Error),
+    #[error(transparent)]
+    Rpc(#[from] soroban_rpc::Error),
+    #[error("unexpected contract data {0:?}")]
+    UnexpectedContractToken(ContractDataEntry),
+    #[error(
+        "cannot fetch wasm for contract because the contract is \
+    a network built-in asset contract that does not have a downloadable code binary"
+    )]
+    ContractIsStellarAsset(),
```

Comment:
Enum values with no arguments don't need to be a tuple enum type.
```suggestion
    ContractIsStellarAsset,
```